package.json

```diff
@@ -47,6 +47,7 @@
     "build:babel": "babel tmp --config-file ./config/babel.config.json --out-dir dist --copy-files",
     "build:cleanup": "rm -rf tmp",
     "lint": "eslint . --ext .ts,.tsx",
+    "lint:fix": "eslint . --ext .ts,.tsx --fix",
```

Comment:
```suggestion
    "lint:fix": "yarn lint:fix --fix",
```

Does that work?

Reply:
Isn't it `yarn lint --fix`?

Reply:
I tried that, but I think that passes the `--fix` flag to `yarn` rather than `eslint`.

Reply:
Right, I think you need the `--`, as in `yarn lint -- --fix`.

Reply:
Yup, you're right, good shout. Updated.